google-cloud-video_intelligence/lib/google/cloud/video_intelligence/v1beta1.rb

```diff
@@ -13,13 +13,15 @@
 # limitations under the License.
 
 require "google/cloud/video_intelligence/v1beta1/video_intelligence_service_client"
+require "google/cloud/videointelligence/v1beta1/video_intelligence_pb"
+require "google/rpc/status_pb"
 
 module Google
   module Cloud
     # rubocop:disable LineLength
 
     ##
-    # # Ruby Client for Google Cloud Video Intelligence API ([GA](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
+    # # Ruby Client for Google Cloud Video Intelligence API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
```

Comment:
Video Intelligence is `GA`. Why is it being downgraded to `Alpha`?

Reply:
@jbolinger Do you know why this is? It's happening for the v1beta1 and v1beta2 clients but not the v1 client.

Reply:
@dazuma it's controlled by a setting (Alpha is the default value). See https://github.com/googleapis/googleapis/blob/ce7d34ee23cf759b9c323d902a067ea4bd1537f4/google/cloud/videointelligence/v1/videointelligence_gapic.yaml#L14

Reply:
@jbolinger That file appears to be generated as well. Do you know what the source of truth is?

Reply:
@dazuma yes. Sorry, I assumed you were familiar with that part of the process. I will ping you directly with the details.

Reply:
Thanks @jbolinger for the info. @blowmage I will get this fixed upstream and will update this PR. Slowly figuring out the process... :)

Reply:
Committed a fix.